README.md

```diff
@@ -27,6 +27,76 @@ https://eui-event.vercel.app/
 - Run `yarn start dev` to start it up!
 - Go to localhost:3000 in your browser to view it!
 
+## Connect Google Sheets with your app
+
+Follow these steps to connect your google sheets file with the app.
+
+### 1. Create a google sheet
+
+#### 1. Go to https://docs.google.com/ and create a blank sheet
+#### 2. Add name, title, team, location, shortBio, pronouns, imageLink headers in the sheet.
+#### 3. Populate the sheet with some data.
+#### 4. Change the sheet name to "Speakers" (Name is case sensitive).
```

Comment:
I think it's technically the tab name, right?

Reply:
Also, is the `name` field the only one that's case sensitive? Don't all the titles need to be lower case and camelCase if more than one word?

Reply:
Actually by "Name is case sensitive", I meant that the Sheet Name "Speakers" is case sensitive. If you change the sheet name to "speakers" then the code would not work.

I will replace "Name is case sensitive" with "Sheet Name is case sensitive" so that user can understand better.
Also the order of headers (title, team, location, shortBio, pronouns, imageLink) should be same too.

So I will add both of these points.

Reply:
ah that makes sense! yes please call that, thanks!